src/pwcode/main.py

```diff
@@ -62,6 +62,13 @@ def latest():
     return data
 
 
+@app.get("/status", summary="Health check", status_code=200)
+def status():
+    endpoint_status = 'Endpoint healthy, 200 OK'
+    logger.info(f'{status}', filename)
```

Comment:
This log message is invalid. The scope of filename variable is local to latest function. Also `{status}` must be `{endpoint_status}`. Also, I suggest to use format syntax (%s) instead of f-strings in logging.

> because internally the logging package uses %-formatting to merge the format string and the variable arguments.

https://docs.python.org/3/howto/logging-cookbook.html#using-particular-formatting-styles-throughout-your-application

It should be:
```
logger.info("%s", endpoint_status)
```
or
```
logger.info(endpoint_status)
```

Also, you can set summary as "Status" and endpoint_status as "OK" (since it already returns HTTP response 200).
Besides this, everything else appears to be good.